src/SignalR.Orleans/Core/IConnectionGroupGrain.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Orleans;
+using System.Threading.Tasks;
+
+namespace SignalR.Orleans.Core
+{
+    public interface IConnectionGroupGrain : IGrainWithStringKey
+    {
+        Task AddMember(string connectionId);
```

Comment:
Groups of any kind (user or group) are mostly connection-oriented so I think it should be better if we rename it to  `AddConnection`. Just a NIPT but keeps the clarity of what it mean since this is base for all types of grains of this kind.

Reply:
Renamed to `Add` since now its `ConnectionGrain`